core/setup.py

```diff
@@ -43,7 +43,7 @@ def create_version_file():
 MIN_PYTHON_VERSION = '>=3.6.*'  # TODO v0.1: Also specify max version? (We don't support 3.8)
 
 requirements = [
-    'numpy>=1.16.0,<1.20.0',
+    'numpy>=1.16.0',
```

Comment:
Will this still work if numpy is 1.16.0?

Reply:
I think we need to fix it to 1.20.x, 1.16 has compatibility issues with a combination of `ConfigSpace` and `pyarrow`